plugins/glustershd/init.go

```diff
@@ -29,11 +29,18 @@ func (p *Plugin) RestRoutes() route.Routes {
 			Pattern:     "/volumes/{name}/heal/enable",
 			Version:     1,
 			HandlerFunc: glustershEnableHandler},
+		route.Route{
+			Name:        "GlustershDisable",
+			Method:      "POST",
+			Pattern:     "/volumes/{name}/heal/disable",
+			Version:     1,
+			HandlerFunc: glustershDisableHandler},
```

Comment:
Does this PR also support disabling/enabling shd on individual volumes (in a cluster where there is more than 1 volume)? From my understanding, the handlers listed above are for stopping or starting the daemon as a whole.

Reply:
Yeah, I have added a flag HealFlag in volinfo struct, so whenever there is a situation in which I need to disable self heal only on lets say 1 volume and not on other volumes I will mark that flag as false for that volume so while running the heal operation on that volume I will check for this flag first.